openhands/integrations/github/service/resolver.py

```diff
@@ -120,7 +120,7 @@ async def get_review_thread_comments(
                 'first': 50,
             }
             if after_cursor:
-                threads_variables['after'] = after_cursor
+                threads_variables['after'] = after_cursor  # type: ignore[unreachable]
```

Comment:
My *guess* is it's wrong about this being unreachable because of it doesn't know the type of page_info